src/ringmesh/mesh/mesh.cpp

```diff
@@ -43,24 +43,25 @@
 #include <stack>
 
 #include <ringmesh/mesh/mesh_index.h>
+#include <ringmesh/mesh/mesh_base.h>
 
 namespace RINGMesh
 {
-    template < index_t DIMENSION >
-    const NNSearch< DIMENSION >& MeshBase< DIMENSION >::vertex_nn_search() const
-    {
-        if( !vertex_nn_search_ )
-        {
-            std::vector< vecn< DIMENSION > > vec_vertices( nb_vertices() );
-            for( auto v : range( nb_vertices() ) )
-            {
-                vec_vertices[v] = vertex( v );
-            }
-            vertex_nn_search_.reset(
-                new NNSearch< DIMENSION >( vec_vertices, true ) );
-        }
-        return *vertex_nn_search_.get();
-    }
+//    template < index_t DIMENSION >
+//    const NNSearch< DIMENSION >& MeshBase< DIMENSION >::vertex_nn_search() const
+//    {
+//        if( !vertex_nn_search_ )
+//        {
+//            std::vector< vecn< DIMENSION > > vec_vertices( nb_vertices() );
+//            for( auto v : range( nb_vertices() ) )

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this block of commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3ACommentedCode)